INCWaveMountainsView/Classes/INCWaveMountainsView.m

```diff
@@ -0,0 +1,400 @@
+//
+//  INCWaveMountainsView.m
+//  eDetailing
+//
+//  Created by Carlos Pages on 26/01/2017.
+//  Copyright © 2017 IncunaLTD. All rights reserved.
+//
+
+#import "INCWaveMountainsView.h"
+#import "INCWaveMountainLayer.h"
+
+typedef NS_ENUM(NSInteger, INCMountainPosition){
+    INCMountainPositionLeft,
+    INCMountainPositionCenter,
+    INCMountainPositionRight
+};
+
+@interface INCWaveMountainsView ()<INCWavwMountainLayerDelegate>
+
+
+//Left mountain
+@property(nonatomic,strong)INCWaveMountainLayer *leftMountain;
+
+
+//Central mountain
+@property(nonatomic,strong)INCWaveMountainLayer *centerMountain;
+
+//Right mountain
+@property(nonatomic,strong)INCWaveMountainLayer *rightMountain;
+
+
+@property(nonatomic,strong)NSMutableDictionary *percentReachFull;
+@property(nonatomic,strong)UIBezierPath *backgroundPath;
+
+@end
+
+
```

Comment:
Those lines doesn't exist in the next pr https://github.com/incuna/INCWaveMountainsView/pull/3